jest/helpers/consoleWarnMock.ts

```diff
@@ -0,0 +1,10 @@
+export default function () {
```

Comment:
looks great!! let's leave a comment here including:
- Purpose of the helper
- How to use the helper
- Any notes (e.g. can't be inside an `it` block, needs to be at a top-level)

Reply:
Done

Reply:
🔥  documentation